test/tools/libtesteth/ImportTest.cpp

```diff
@@ -629,6 +654,10 @@ int ImportTest::exportTest(bytes const& _output)
 				if (it != std::end(stateIndexesToPrint))
 					obj2["postState"] = fillJsonWithState(m_transactions[i].postState);
 			}
+
+			if (Options::get().statediff)
```

Comment:
Do we really need to write it into files and incerase their size and complexity? It's only for human readers, right?

Maybe just leave it like a log option for `testeth` run ?